### PR TITLE
Update What Is Kontena? documentation

### PR DESCRIPTION
**Pull request by nevalla**
This PR changes 'Kontena system' to 'Kontena Platform'. Also added mention about Kontena Cloud.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Kontena's key features include:
 * Log and statistics aggregation with streaming
 * Access control and roles for Kontena users
 
-Kontena is administered via the Kontena command line interface, **Kontena CLI**. At the moment, there is no graphical UI for Kontena.
+Kontena is administered via the Kontena command line interface, **Kontena CLI**. **[Kontena Cloud](https://cloud.kontena.io)** provides hosted Graphical UI for monitoring, analyzing and operating Kontena Platforms.
```
Comment (kke) [inline docs/README.md]: "provides `a` hosted.".

maybe also "Graphical UI" -> "(?graphical) user interface".

Also looks like `command-line interface` comes with a hyphen, for example in (Wikipedia)[https://en.wikipedia.org/wiki/Command-line_interface]

File: docs/README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ toc_order: 1
 
 # What is Kontena?
 
-[Kontena](http://www.kontena.io) is an open source project for orchestrating and running containerized workloads on a cluster. The Kontena system is comprised of a number of **Kontena Nodes** (meaning servers or virtual machines that run containerized workloads) and a **Kontena Master** (which controls and monitors the Nodes).
+[Kontena](http://www.kontena.io) is an open source project for orchestrating and running containerized workloads on a cluster. The **Kontena Platform** is comprised of a number of **Kontena Nodes** (meaning servers or virtual machines that run containerized workloads) and a **Kontena Master** (which controls and monitors the Nodes).
 
 With Kontena, you can construct your application using the **Kontena Service** definition. A Service definition describes the container image, networking, scaling and stateful/stateless attributes of your application. Services may be linked together to create the desired architecture. Each service is automatically assigned an internal DNS address, which can be used inside your application for inter-Service communications.
```
Comment (kke) [inline docs/README.md]: Should this say something about stacks?

File: docs/README.md (hunk 1 above)
Comment (SpComb) [inline docs/README.md]: ping @nevalla

Comment (jakolehm) [inline docs/README.md]: Link to Kontena Cloud should be https://www.kontena.io/cloud

**Comment by nevalla** [comment]
Added missed article and some fine tuning. Also updated the link to Kontena Cloud.

**Pull request merged**